src/sagemaker/serve/builder/transformers_builder.py

```diff
@@ -301,6 +348,27 @@ def _build_for_transformers(self):
         self.secret_key = None
         self.model_server = ModelServer.MMS
 
+        if self.inference_spec:
+
+            if not os.path.exists(self.model_path):
+                os.makedirs(self.model_path)
+
+            code_path = Path(self.model_path).joinpath("code")
+
+            save_pkl(code_path, (self.inference_spec, self.schema_builder))
+            logger.info("PKL file saved to file: %s", code_path)
+
+            self._auto_detect_container()
```

Comment:
nit: could you cover these lines with unit tests? this will increase code coverage.